src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -57,7 +57,9 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
         validationDelayMs: PASSWORD_REQUIREMENT_VALIDATION_DELAY_MS,
       },
     };
-    uischema.elements.unshift(passwordRequirementsElement);
+    if (Object.keys(passwordSettings)?.length) {
+      uischema.elements.unshift(passwordRequirementsElement);
+    }
```

Comment:
Yea, so what was happening was it would add an empty element when the password settings were not provided. Visually, it caused a minor empty margin, this just gets rid of that since there is nothing to actually display.